EcoSoapBankTests/GraphQLMutationTests.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  GraphQLMutationTests.swift
+//  EcoSoapBankTests
+//
+//  Created by Christopher Devito on 8/22/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import XCTest
+@testable import EcoSoapBank
+
+class GraphQLMutationTests: XCTestCase {
+
+    func testLogin() {
+        guard let path = Bundle.main.path(forResource: "mockUserByIdInput",
+                                          ofType: "json"),
+            let mockData = NSData(contentsOfFile: path) else {
+                XCTFail("Unable to get mock impact stats data from path")
+                return
+        }
+        let data = Data(mockData)
+        let mockLoader = MockDataLoader(data: data,
+                                        error: nil)
+        let graphQLController = GraphQLController(session: mockLoader)
```

Comment:
Not a huge deal since there's only two tests, but if you write more it might be worth pulling some of this out into helper properties/functions/`setUp`.

Reply:
Good Idea. I'll work on that for my next PR since I definitely have more tests coming.